### PR TITLE
Feature: free form `--help` text

### PR DESCRIPTION
**Pull request by jodosha**

### CODE REVIEW

**Comment by rngtng** [comment]
oh nice, I was looking exactly for this.. what's the status?

File: lib/dry/cli/command.rb [hunk 1]
```diff
@@ -108,6 +109,46 @@ def self.example(*examples)
...
+      # @param blk [Proc] a block that must return a free form string to add
+      #   extra information
+      #
+      # @since 0.8.0
+      #
+      # @example
+      #   require "dry/cli"
+      #
+      #   class Convert < Dry::CLI::Command
+      #     help do
```
Comment (rngtng) [inline lib/dry/cli/command.rb:123]: Have you though to support just `help "String"` too?

Comment (jodosha) [reply lib/dry/cli/command.rb:123]: @rngtng Yeah, that could work too.

**Comment by jodosha** [comment]
@rngtng The project where I'm using this feature is low priority. I want this feature to be driven by real world needs, that's why this PR is not finished yet.

**Pull request closed without merging**